### PR TITLE
adds mailjet, fixes a few mailgun issues

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: appengine/standard/mailjet/app.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+ * limitations under the License.
+ */
+
+use Silex\Application;
+
+// create the Silex application
+$app = new Application();
+
+$app->get('/', function () use ($app) {
+    if ($app['mailjet.api_key'] == 'YOUR_API_KEY') {
```
Comment (tmatsuo) [inline appengine/standard/mailjet/app.php]: Nit: `YOUR_API_KEY` looks too generic.

File: appengine/standard/mailjet/index.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+// Install composer dependencies with "composer install"
+// @see http://getcomposer.org for more information.
+require __DIR__ . '/vendor/autoload.php';
+
+$app = require __DIR__ . '/app.php';
+
+// set your Mailjet API key and secret
+$app['mailjet.api_key'] = 'YOUR_API_KEY';
+$app['mailjet.secret'] = 'YOUR_SECRET';
```
Comment (tmatsuo) [inline appengine/standard/mailjet/index.php]: NIT: `YOUR_SECRET` looks very generic, although I can live with this.

Comment (bshaffer) [inline appengine/standard/mailjet/index.php]: what would you prefer?

Comment (tmatsuo) [inline appengine/standard/mailjet/index.php]: How about `MAILJET_API_KEY` and `MAILJET_SECRET`?

**Comment by JustinBeckwith** [comment]
LGTM

File: appengine/standard/mailjet/index.php (hunk 2 above)
Comment (bshaffer) [inline appengine/standard/mailjet/index.php]: Done. Fixed in `mailgun` sample as well.

**Pull request merged**